x-pack/plugins/cases/server/client/cases/create.test.ts

```diff
@@ -1,464 +0,0 @@
-/*
```

Comment:
were these deleted test files moved?

Reply:
To quote @jonathan-buttner 

> we actually removed a lot of the unit tests that were mocking different parts of the cases implementation because we've found that it was hard to maintain them as we were restructuring code and changing the implementation. We moved most of the mocked logic into integration tests if they didn't already exist.